group_project/group_activity.py

```diff
@@ -477,4 +477,3 @@ def import_xml_file(cls, file_name):
     def import_xml_string(cls, xml, grading_override = False):
         doc_tree = ET.fromstring(xml)
         return cls(doc_tree, grading_override)
-
```

Comment:
I have a pep8 validator in sublime and it marked that extra line as an error line. That's the reason behind the removal.

I did extra testing by running the `pep8 group_activity.py` and it pep8 seems ok with the removal of this line. Github doesn't show any of the empty lines at the end of the file, so it seems that the only empty line is removed here.